src/main/java/org/manifold/compiler/back/digital/smt2/ParenList.java

```diff
@@ -0,0 +1,44 @@
+package org.manifold.compiler.back.digital.smt2;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
+public class ParenList extends SExpression {
+
+  private List<SExpression> exprs = new ArrayList<>();
+  public List<SExpression> getExprs() {
+    return ImmutableList.copyOf(exprs);
+  }
+  
+  public ParenList() { }
+  
+  public ParenList(SExpression expr) {
+    this.exprs.add(expr);
+  }
+  
+  public ParenList(SExpression exprs[]) {
```

Comment:
Might be simpler to make a constructor taking a variable number of SExpressions, especially in Symbol
```java
public ParenList(SExpression ... exprs)
```